src/OCPI.Net.Contracts/ChargingProfiles/OcpiSetChargingProfileCommand.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text.Json.Serialization;
+
+namespace OCPI.Contracts.ChargingProfiles;
+public class OcpiSetChargingProfileCommand
```

Comment:
I understand where the idea of calling it a `command` comes from but I am worried about it potentially creating confusion between the commands featured in the Commands Module and this. How about we name
 it `OcpiSetChargingProfileRequest` instead?

Reply:
done